creating_certificates.md

```diff
@@ -65,11 +65,13 @@ Example of a valid `issuedOn` string is `2018-08-31T23:59:32+08:00`. Where it sp
 
 ### `issuers` field
 
-The `issuers` is an array of the issuer object. The required fields for each issuer is `name` and `certificateStore`. Other fields such as `did`, `url`, `email` and `phone` are also recognised.
+The `issuers` is an array of the issuer object. The required fields for each issuer is `name`and `documentStore`. Other fields such as `did`, `url`, `email`, `identityProof` and `phone` are also recognised.
 
-The `name` specifies the name of the issuing body while the `certificateStore` specifies the smart contract address of the certificate store deployed by the issuing body.
+The `name` specifies the name of the issuing body while the `documentStore` specifies the smart contract address of the document store deployed by the issuing body.
 
-The `certificateStore` address can be obtained after [deploying an instance of the smart contract](./deploying_store.md).
+The `documentStore` address can be obtained after [deploying an instance of the smart contract](./deploying_store.md).
+
+Even if `identityProof` is not required, it's highly recommended to enable it on every certificate. More information about [DNS verification](./dns_verification.md).
```

Comment:
Will you like to specify that `identityProof` is not required in OC v2 for legacy reason. To ensure valid that your OpenCerts document fully conforms to the OpenAttestation v2 Standards, and therefore allowed to be rendered on OA compatible platforms, `identityProof` is required.

Reply:
good idea